TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -73,10 +84,16 @@ class AudioRecordingsTableViewController: UITableViewController {
                     
                     self.tableView.reloadData()
                 }
+                
             }
             
-            ac.addAction(renameFileAction)
-            self.present(ac, animated: true)
+            let cancelEditAction = UIAlertAction(title: "Cancel", style: .cancel, handler: { _ in
+                completionHandler(false)
```

Comment:
You can leave the completion block empty here or mark it as nil

Reply:
Would I still be able to set the completionHandler as false? I needed this to get rid of the UIConextualAction UI after selecting cancel.

Reply:
Ah ok makes sense then :D